### ISSUE TITLE
Layout/CaseIndentation: the name of rule is not printed in description

### ISSUE THREAD

**Issue by deepj** [rubocop-hq/rubocop#8764]
When `Layout/CaseIndentation` is detected, its name is not printed in description. Other rules it have in their description.

## Expected behavior

```
app/helpers/test_helper.rb:25:7: C: Layout/CaseIndentation: Indent when as deep as case.
    when :small then 16
    ^^^^
```

## Actual behavior

```
app/helpers/test_helper.rb:25:7: C: Indent when as deep as case.
    when :small then 16
    ^^^^
```

## Steps to reproduce the problem

```ruby
    w_h = case size
      when :small then 16
      when :big then 32
      when :huge then 64
      else 24
          end
```

Run

```
rubocop --only Layout/CaseIndentation
```

## RuboCop version

```
$ rubocop -V
0.91.0 (using Parser 2.7.1.4, rubocop-ast 0.4.0, running on ruby 2.7.1 x86_64-darwin18)
```
